src/fileoverview_comment_transformer.ts

```diff
@@ -22,13 +23,20 @@ const FILEOVERVIEW_COMMENT_MARKERS: ReadonlySet<string> =
  * Given a parsed \@fileoverview comment, ensures it has all the attributes we need.
  * This function can be called to modify an existing comment or to make a new one.
  *
+ * @param host Processor host to calculate relative TS file path.
+ * @param source Original TS source file. Its path is added in \@fileoverview.
  * @param tags Comment as parsed list of tags; modified in-place.
  */
-function augmentFileoverviewComments(tags: jsdoc.Tag[]) {
+function augmentFileoverviewComments(
+    host: GoogModuleProcessorHost, source: ts.SourceFile, tags: jsdoc.Tag[]) {
   // Ensure we start with a @fileoverview.
   if (!tags.find(t => t.tagName === 'fileoverview')) {
     tags.splice(0, 0, {tagName: 'fileoverview', text: 'added by tsickle'});
   }
+  const fileOverview = tags.find(t => t.tagName === 'fileoverview');
+  if (fileOverview) {
+    fileOverview.text += `\nGenerated from: ${host.fileNameToModuleId(source.fileName)}`;
```

Comment:
that's some really bad API naming on our side here (module ID turns out to be the path?). Nothing to fix in this CL.

Reply:
I think module id happens to be the path, but it's not part of its guarantee.

Node docs say "The identifier for the module. Typically this is the fully resolved filename."
https://nodejs.org/api/modules.html#modules_module_id

For example, does this include blaze-bin or not?  Should it?  Which behavior do we depend on in both of its use cases?

Reply:
I'll see how it works in google3. I kinda suspect it won't work as expected.

Reply:
Following Evan's suggestion, changed code to use `CompilerOptions.rootDir` to produce root-relative paths. Checked and it works with google3 generated .ts files as well.